### PR TITLE
fixed bugs, fixed flow of auth and fixed edit profile for company

### PR DESCRIPTION
**Pull request by subru-37**

### CODE REVIEW

**Review by jaison080 — APPROVED** [review]

File: src/Components/CustomTextfield/CustomTextfield.jsx [hunk 1]
```diff
@@ -22,7 +22,10 @@ const CustomTextfield = ({
   minRows
 }) => {
   function handleChange(event) {
-    const { name, value } = event.target;
+    var { name, value } = event.target;
+    // if (name === "state" || name === "district"){
```
Comment (jaison080) [inline src/Components/CustomTextfield/CustomTextfield.jsx]: Remove comments if not neccessary.

File: src/Components/NavbarHome/NavbarHome.js [hunk 2]
```diff
@@ -37,9 +37,9 @@ function NavbarHome() {
           <Link to="/ideas" className="nav_item">
             Ideas
           </Link>
-          <Link to="/jobs" className="nav_item">
+          {/* <Link to="/jobs" className="nav_item">
```
Comment (jaison080) [inline src/Components/NavbarHome/NavbarHome.js:40]: Remove comments if not neccessary.

File: src/Components/NavbarHome/NavbarHome.js [hunk 3]
```diff
@@ -105,13 +105,13 @@ function NavbarHome() {
               >
                 Ideas
               </Link>
-              <Link
+              {/* <Link
```
Comment (jaison080) [inline src/Components/NavbarHome/NavbarHome.js:108]: Remove comments if not neccessary.

File: src/Components/OrganizationForm/OrganizationForm.jsx [hunk 4]
```diff
@@ -26,7 +28,22 @@ const OrganizationForm = ({ openModal, setOpenModal }) => {
         autoClose: 2000,
       });
     });
-
+    history.push("/profile");
+    setCompanyDetails({
+      description: "",
```
Comment (jaison080) [inline src/Components/OrganizationForm/OrganizationForm.jsx:32]: Remove comments if not neccessary.

File: src/Pages/Jobs/Jobs.scss [hunk 5]
```diff
@@ -9,15 +9,15 @@
...
-  text-align: center;
-  font-style: normal;
-  font-weight: 800;
-  font-size: 30px;
-  line-height: 40px;
-  padding-top: 5rem;
-  color: #9e0000;
-}
+// .developer-title {
+//   text-align: center;
```
Comment (jaison080) [inline src/Pages/Jobs/Jobs.scss]: Remove comments if not neccessary.

File: src/Components/CustomTextfield/CustomTextfield.jsx (hunk 1 above)
Comment (subru-37) [reply src/Components/CustomTextfield/CustomTextfield.jsx]: this comment is not necessary

File: src/Components/NavbarHome/NavbarHome.js (hunk 3 above)
Comment (subru-37) [reply src/Components/NavbarHome/NavbarHome.js:108]: this comment is necessary, the route exists, but page is not ready so as soon as the page is ready, ithu uncomment chethal mathi

**Pull request merged**